var/spack/repos/builtin.mock/packages/boost/package.py

```diff
@@ -61,4 +61,5 @@ class Boost(Package):
             description="Augment library names with build options")
 
     def install(self, spec, prefix):
-        pass
+        # sanity_check_prefix requires something in the install directory
+        mkdirp(prefix.include)
```

Comment:
why include here and bin elsewhere?

Reply:
To reflect that Boost is a library.  I should've taken the time to check whether it produces a `bin` directory.

Reply:
That seems sort of beside the point for testing — unless something specifically needs a different file in the install dir to mock an install, I’d consolidate this to a common TestPackage or MockPackage base class and be consistent rather than emphasizing something we don’t really use functionally in the tests.

Reply:
Will refactor per your feedback for `hg-top-level`.